addons/web/static/tests/legacy/fields/relational_fields_tests.js

```diff
@@ -1019,6 +1025,48 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    // TODO: Once the code base is converted with wowl, replace webclient by formview.
+    QUnit.test('statusbar edited by the smart action "Move to stage..."', async function (assert) {
+        assert.expect(3);
+
+        const legacyEnv = makeTestEnvironment({ bus: core.bus });
+        const serviceRegistry = registry.category("services");
+        serviceRegistry.add("legacy_command", makeLegacyCommandService(legacyEnv));
+
+        const views = {
+            'partner,false,form': '<form>' +
+                    '<header><field name="trululu" widget="statusbar" options=\'{"clickable": "1"}\'/></header>' +
+                  '</form>',
+            'partner,false,search': '<search></search>',
+        };
+        const serverData = { models: this.data, views}
+        const webClient = await createWebClient({serverData});
+        await doAction(webClient, {
+            res_id: 1,
+            type: 'ir.actions.act_window',
+            target: 'current',
+            res_model: 'partner',
+            'view_mode': 'form',
+            'views': [[false, 'form']],
+        });
+        assert.containsOnce(webClient, ".o_field_widget")
+
+        triggerHotkey("control+k")
+        await nextTick();
+        const idx = [...webClient.el.querySelectorAll(".o_command")].map(el => el.textContent).indexOf("Move to Trululu...ALT + SHIFT + X")
+        assert.ok(idx >= 0);
+
+        await click([...webClient.el.querySelectorAll(".o_command")][idx])
```

Comment:
With `NodeList`, we can select the element with an id like an `Array`. So, normally it should also work like this: 
```suggestion
        await click(webClient.el.querySelectorAll(".o_command")[idx])
```
🙂